app/internal/page/certificateprovider/your_details.go

```diff
@@ -99,18 +90,24 @@ func (f *yourDetailsForm) DobWarning() string {
 func (f *yourDetailsForm) Validate() validation.List {
 	var errors validation.List
 
-	errors.Date("date-of-birth", "dateOfBirth", f.Dob,
+	errors.Date("date-of-birth", "yourDateOfBirth", f.Dob,
 		validation.DateMissing(),
-		validation.DateMustBeReal(),
 		validation.DateMustBePast())
 
-	errors.String("mobile", "mobile", strings.ReplaceAll(f.Mobile, " ", ""),
-		validation.Empty(),
-		validation.Mobile())
+	if !f.Dob.Valid() {
+		errors.Add("date-of-birth", validation.EnterError{Label: "aValidDateOfBirth"})
+	}
+
+	if f.Dob.After(date.Today().AddDate(-18, 0, 0)) {
+		errors.Add("date-of-birth", validation.CustomError{Label: "youAreUnder18Error"})
+	}
+
+	errors.String("mobile", "yourUkMobile", strings.ReplaceAll(f.Mobile, " ", ""),
+		validation.Empty())
```

Comment:
can you just use `validation.Mobile()`, I know they've said to use specific wording but I'd rather we keep it consistent (and so change the `errorMobile` translation to make sense if necessary)

Reply:
I was going to look at the way we're doing validation errors as a whole once we've got everything ready for UR as it would be good to have consistency throughout the whole app so we can standardise all of this in the validation package.

Worth sitting down with content and design to agree this and then we can make updates where needed.